### DISCUSSION TITLE
Modifying sliders in editor resets the hitsound type and volume.

### DISCUSSION THREAD

**Discussion by Coppertine** [ppy/osu#15615] [Q&A] [answered]
Self explanitory, happens on any standard beatmap in the editor. This happens when you move any anchor, press CTRL+G, using the Rotate buttons, using the Horizontal or vertical flips, or moving the slider on the playfield.
My guess is that the slider is replaced by a default slider and does not respect the current hitsounds on said slider.

**Comment by bdach** [comment, marked as answer]
#15618
